<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_back">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>‚Üê zur√ºck</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_inputs">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Eingaben:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_inputs">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;</string>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_remove_selected_items">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>üóë</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_total_title">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Gesamt:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_total_amount">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>0,00 ‚Ç¨</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
         <item>
          <widget class="QLabel" name="label_amount_input">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Betrag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_input">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_9">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="pushButton_clear">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_8">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_0">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButton_comma">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>,</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButton_del">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>‚Üê</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QPushButton" name="pushButton_add">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_1">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save_back">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>speichern &amp;&amp; zur√ºck</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
